cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/SessionConfiguration.java

```diff
@@ -1,15 +1,15 @@
 /*
- * Copyright (c) 2016 - 2017 Memorial Sloan-Kettering Cancer Center.
+ * Copyright (c) 2016, 2017, 2023 Memorial Sloan Kettering Cancer Center.
```

Comment:
I assume we are not including years where the file wasn't modified.

Reply:
Basically yes, although I did not always check the git history to see which years modifications actually happened in. We should probably switch to using a project-wide copyright header which is identical across all source files. If any modification to any file occurs within a year, then all headers across the project will be updated even for individual files which did not change. Authors revise books, and they do not track changes to individual chapters (or pages). Possibly we could have a uniform copyright header per-module or other distributable artifact.